keychain/vault/vault_test.go

```
@@ -22,3 +27,105 @@ func TestNewKeyGroup(t *testing.T) {
+			if err := json.Unmarshal(body, &req); err != nil {
+				http.Error(w, err.Error(), http.StatusBadRequest)
+				return
+			}
+			// Just prepend "vault:v1:" to the base64 plaintext to simulate ciphertext
+			resp := map[string]interface{}{
+				"data": map[string]string{
+					"ciphertext": "vault:v1:" + req.Plaintext,
+				},
+			}
+			respBytes, _ := json.Marshal(resp)
+			w.Write(respBytes)
```

golangci-lint (2.12.2)

[error] Error return value of `w.Write` is not checked

(errcheck)